schema/playground/spawner.schema

```diff
@@ -1,20 +0,0 @@
-package playground;
```

Comment:
why is spawner schema removed here?

Reply:
This is a drive-by fix. The component was used in the past for creating the player entity but that has been moved to a different component now in the player lifecycle module. The spawner component isn't being used anymore so I deleted it.

Reply:
👍